src/components/dialogs/NotificationDetailsDialog.jsx

```diff
@@ -38,7 +40,10 @@ export default function NotificationDetailsDialog({
     theirIndividualName,
     theirIndividualGuid,
     formattedDeadline,
-  } = getNotificationProps(intl, notification);
+    otherUserGuidForManagerNotifications,
+    otherUserNameForManagerNotifications,
+    managerName,
+  } = getNotificationProps(intl, notification, currentUserData?.guid);
```

Comment:
If we want error and loading to be tracked for useGetMe, how would I best place this getNotificationProps behind a logic barrier that required loading and error be false?

Reply:
Here's one idea:
```
const props = (isLoading || error) ? {} : getNotificationProps(intl, notification, currentUserData?.guid);
```

Reply:
Ok I implemented this in [d0f5ef0](https://github.com/WildMeOrg/codex-frontend/pull/463/commits/d0f5ef001f8830958c282db36d705c3cccec2779). Seems to have worked. Thanks!